configuration/configurator/ConfiguratorAPI.hpp

```diff
@@ -0,0 +1,193 @@
+#ifndef _CONFIGURATOR_API_HPP_
+#define _CONFIGURATOR_API_HPP_
+
+#define MAX_ARRAY_X 100
+#define MAX_ARRAY_Y 128
+
+const char modelNames[MAX_ARRAY_X][MAX_ARRAY_Y] =
+                                        {   "tableDataModel0",
```

Comment:
This does not feel like the right way to do whatever it is you are trying to do here...